src/Symfony/Component/RateLimiter/FixedWindowLimiter.php

```diff
@@ -24,10 +24,29 @@
  */
 final class FixedWindowLimiter implements LimiterInterface
 {
+    /**
+     * @var string
+     */
     private $id;
+
+    /**
+     * @var int
+     */
     private $limit;
+
+    /**
+     * @var \DateInterval
```

Comment:
```suggestion
     * @var int
```

Reply:
Yes, You are correct. I missed that we converted the DateInterval to an int in the constructor. 

Is this proof that annotations is really useful? =)